package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please remove this file, it is not needed for Arduino cores.

Reply:
Will do.